src/services/ScheduleService.ts

```diff
@@ -104,6 +104,20 @@ export class ScheduleService extends Service {
         this.db.getEmitterSet<Schedule>(Schedule.name).update(await this.getAll());
     }
 
+    /**
+     * Gets the {@link Schedule} that is currently enabled, or undefined if no {@link Schedule} is enabled.
+     */
+    public async getEnabled(): Promise<Schedule | undefined> {
```

Comment:
Can we get this added to the mock as well?

Reply:
Will do!